docs/features/particle-system.md

```
@@ -0,0 +1,72 @@
+## Effect controls
+
+Since particle effects are based on physics parameters they can be tuned a lot more than normal effects by using the effect sliders.
+Each effect uses different slider controls such as:
+
+- Gravity strength and direction
+- Particle size
+- Particle lifetime / spawn rate
+- Collision on/off
+- Wrap-around edges on/off
+
+The default slider settings were chosen such that the effect looks nice on many different setups but often can be optimized for a specific situation. A description of what each slider/checkmark does for each effect is described [here](/features/effects).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)